### PR TITLE
[minor] Output divisions email instead of employee in declaration request printout form

### PR DESCRIPTION
**Pull request by progsmile**

### CODE REVIEW

File: priv/repo/migrations/20180507144151_set_divisions_email_in_declaration_request_printout_form.exs [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
...карських засобів,  функціонування електронної системи охорони здоров’я. </div> <div class="small"> Підписуючи цю Декларацію, я надаю згоду на доступ до перегляду даних про мене (пацієнта, законним представником  якого я є), що містяться в електронній системі охорони здоров’я, лікарю, який надає ПМД, зазначеному  в цій Декларації, а також іншим лікарям, які надаватимуть мені (пацієнту, законним представником якого  я є) медичну допомогу, в межах, необхідних для надання такої допомоги. </div> <div>  <div class="small"> Підпис пацієнта (його законного представника) <i>(у разі заповнення у паперовій формі)</i>  </div>  <div class="sign"></div> </div></div><div class="el"> <div class="el__header">  <b>6. Підпис пацієнта (його законного представника)</b> </div> <div class="small"> Своїм підписом підтверджую добровільний вибір лікаря, який надає первинну медичну допомогу, що зазначений  в розділі 2 цієї Декларації, а також достовірність наданих мною даних. </div> <div>  <div class="sign"></div> </div></div><div class="el"> <div class="el__header">  <b>7. Дата подання</b> </div> <div class="el__data">  <%= assigns[:start_date] %> </div></div>  </div> </section></body></html>' WHERE id = 4;
```
Comment (AlexKovalevych) [inline priv/repo/migrations/20180507144151_set_divisions_email_in_declaration_request_printout_form.exs]: No need to set the syntax anymore

Comment (progsmile) [reply priv/repo/migrations/20180507144151_set_divisions_email_in_declaration_request_printout_form.exs]: so, `UPDATE body = '...'`?

Comment (AlexKovalevych) [reply priv/repo/migrations/20180507144151_set_divisions_email_in_declaration_request_printout_form.exs]: yup

Comment (progsmile) [reply priv/repo/migrations/20180507144151_set_divisions_email_in_declaration_request_printout_form.exs]: fixed

**Pull request merged**